build-python.sh

```diff
@@ -19,9 +19,9 @@ set -e
 : ${LIBFFI_VERSION:=v3.4.2}
 : ${PYTHON_MAJOR:=3}
 : ${PYTHON_MINOR:=9}
-: ${PYTHON_PATCH:=6}
+: ${PYTHON_PATCH:=7}
 : ${PYTHON_VERSION:=v${PYTHON_MAJOR}.${PYTHON_MINOR}.${PYTHON_PATCH}}
-: ${PYTHON_VERSION_MINGW:=61c5a4a14932539b31878de0b60bc98bcba791b9}
+: ${PYTHON_VERSION_MINGW:=mingw-${PYTHON_VERSION}}
```

Comment:
Looks great overall, but as said, I'd prefer to keep this at an exact individual commit, for reproducibility reasons.